snap/snapcraft.yaml

```
@@ -223,7 +223,7 @@ parts:
     edge-core:
       plugin: cmake
       source: https://github.com/ARMmbed/mbed-edge.git
-      source-tag: 0.13.0
+      source-tag: 0.15.0
```

```suggestion
      source-tag: 0.16.1
```
@costanic New version of edge-core is available so if we are upgrading then let's go to latest one? Not sure if we will have to regenerate all the patches to work with new version or not.

---

@ygoyal18 This PR, plus an [upcoming PR from the pe-2.2](https://github.com/PelionIoT/snap-pelion-edge/tree/pe-2.2) branch encompasses an effort to upgrade snap to PE2.2.  Is PE2.2 running with 0.16.1?

---

You are right, 2.2 is on EC 0.15. But 2.3 will be available by end of this month and that will take us to 0.16.1. We will again we out of sync and as we haven't yet planned to support Parsec in snap, we won't get chance to upgrade edge-core to latest version. My suggestion is to take latest version if we are in this space and doing the leg work to upgrade edge-core version.

---

@ygoyal18 Do you have a list of other component versions for PE2.3?  I.e., which version of edge-proxy, maestro, pe-utils, dockerd, etc.?  Please see [this branch](https://github.com/PelionIoT/snap-pelion-edge/commits/pe-2.2) for current status of upgrading snap components for 2.2, which was planned to be merged after this PR.  Which of those components will need to be upgraded again for 2.3 and which are already at the correct version?

---

AFAIK, today there is no easy way to get that information out of meta-pelion-edge and meta-mbed-edge. I remember we discussed with @ryannowarm to have a version yaml file or similar in our packages which will give us that information in one shot. I forgot what we called it. 

From comparing the dev with master branch of Yocto Pelion Edge I can only see mbed-edge being upgraded to 0.16.1 and mbed-edge-examples to 0.16.0. Rest all the packages are at the same version as 2.2.

---

ok thanks @ygoyal18, I can just look at the metas until we have a PE version index or whatever we called it.